### DISCUSSION TITLE
微生物是污水處理的關鍵，AI助攻讓水廠淨化更有效率

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#693] [General]
污水處理有三大種類：物理、化學和微生物，其中微生物雖染是最環保的但是卻是技術上來說最難的，因為我們現在對微生物的相關參數認知還是相當有限，為了達到環保的目的，「顯微影像專家、AI新創與水廠」，三方攜手開發出創新解方。

AI新創公司設定了微生物的顯微影像可被AI模型辨識，並且在三個月內不斷輸入資料來訓練模型，且不斷與污水廠合作測試，讓產出的結果更精準。

除了與美髮產業結合，現在又有了與污水處理廠結合的例子，我發現台灣有好多新創的AI公司在默默的為台灣人解決問題或不斷進步，原來我們現在所學的東西真的能應用在未來生活上，而且是如此貼近日常生活。

文章來源：https://www.thenewslens.com/feature/aishow/177007

**Comment by Vincent20030314** [comment]
這項發明很有趣，在解決汙水方面AI確實有大的發展空間。

**Comment by yjpxi** [comment]
通過將顯微影像專家、AI新創公司與水廠的合作，可以提高處理效率，減少對環境的負面影響。
